meerk40t/core/elements.py

```diff
@@ -4459,7 +4459,6 @@ def delete(channel, _, data=None, **kwargs):
                 if n.type not in ("root", "branch elems", "branch ops"):
                     # Cannot delete structure nodes.
                     n.remove_node()
-                    break
```

Comment:
"Delete node. Due to nodes within nodes, only the first node is deleted."

* Break serves the purpose of being lazy.